transformer_engine/pytorch/module.py

```diff
@@ -2224,12 +2226,13 @@ def forward(
                 activation_dtype,
                 get_workspace(),
                 bias=fc1_bias,
-                use_bias=not bias_gelu_nvfusion,
+                use_bias=(not bias_gelu_nvfusion) and use_bias,
```

Comment:
```suggestion
                use_bias=(not bias_gelu_nvfusion) and fc1_bias.numel() > 0,
```

Reply:
Same reason as above comments. However, this is a bit confusing. Switching fully to `use_fc1_bias` and `use_fc2_bias` is better.